src/melee/gr/grdynamicattr.c

```diff
@@ -32,11 +33,16 @@ void func_801CA0B4(void)
     }
 }
 
-UnkStruct* func_801CA0F8(s32 arg0, Vec3* v, s32 floor_id, f32 f, s32 arg3)
+UnkStruct* func_801CA0F8(s32 arg0, Vec3* v, enum_t floor_id, f32 f, s32 arg3)
 {
+#ifdef MUST_MATCH
+    u8 unused[8];
```

Comment:
Why did the padding grow?

Reply:
It didn't, I've been converting them all to `u8` because it's quicker to read.

Reply:
Ah of course